apis/Google.Cloud.Firestore/Google.Cloud.Firestore.IntegrationTests/coverage.xml

```diff
@@ -17,5 +17,5 @@
     <AttributeFilterEntry>System.Diagnostics.DebuggerNonUserCodeAttribute</AttributeFilterEntry>
   </AttributeFilters>
   <TargetWorkingDir>.</TargetWorkingDir>
-  <Output>../../../coverage/Google.Cloud.Firestore.Data.Snippets.dvcr</Output>
+  <Output>../../../coverage/Google.Cloud.Firestore.IntegrationTests.dvcr</Output>
```

Comment:
Is this correct? It's more than just removing ".Data"

Reply:
It's git or github getting confused - it thinks it's a rename from ...Data.Snippets/coverage.xml to ...Data.IntegrationTests/coverage.xml.